lib/ansible/galaxy/role.py

```diff
@@ -30,6 +41,14 @@
 from distutils.version import LooseVersion
 from shutil import rmtree
 
+try:
+    import boto3
+except ImportError:
+    try:
```

Comment:
Yeah, it'd take a couple of those vars due to boto+boto3 support, and not having to check a flag makes the actual usage faster (assuming that boto3 is favored over boto).